plugins/qtAccessionPlugin/lib/model/QubitAccessionEvent.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+
```

Comment:
One extra empty line.

Reply:
Also, missing the ` * This file is part of the Access to Memory (AtoM) software...` comment.